src/Header.cpp

```diff
@@ -20,66 +16,41 @@ static inline std::string trim(const std::string &s)
 	return s.substr(start, end - start + 1);
 }
```

Comment:
This could work with string_view so that caller doesn't need to allocate memory yet.